### PR TITLE
Turn off debug mode for Google Analytics

### LINKED ISSUE #1569: Turn off debug mode for Google Analytics
When we first implemented GA for Code Manager, debug mode was enabled for local development by default. This provides a lot of information that can be useful when working directly with GA, but the amount of info it adds is actually quite annoying when you don't really care about GA.

This issue is about changing the default to be `debug: false`, but allowing it to be changed locally via `.env.common-local`.

For QA:
There will not be any observable differences from this change on any of the hosted environments, but please check that the existing code that sends pings to Google Analytics remains unaffected.

### PR DESCRIPTION
**Pull request by bobsilverberg**
Fixes #1569 

I also found a bug in the way we were handling boolean values from `env` files in `tracking.tsx` which I fixed.

### CODE REVIEW

File: .env.common-local [hunk 1]
```diff
@@ -26,3 +26,6 @@ PUBLIC_URL=/
 # Variables for Google Analytics
 REACT_APP_TRACKING_ENABLED=false
 REACT_APP_TRACKING_ID=UA-77033033-31
+
+# This is used for local development to toggle debug mode for Google Analytics
+REACT_APP_GA_DEBUG_MODE=false
```
Comment (eviljeff) [inline .env.common-local:31]: Does it need to be redefined in this file? (`.env.common-local` inherits from `.env`?  Or is there an intermediate file where it's `true`?)

File: src/tracking.tsx [hunk 2]
```diff
@@ -46,7 +46,7 @@ export class Tracking {
   constructor({
     _isDoNotTrackEnabled = isDoNotTrackEnabled,
     _reactGA = ReactGA,
-    trackingEnabled = Boolean(process.env.REACT_APP_TRACKING_ENABLED),
+    trackingEnabled = process.env.REACT_APP_TRACKING_ENABLED === 'true',
```
Comment (eviljeff) [inline src/tracking.tsx:49]: curiosity: why does `REACT_APP_TRACKING_ENABLED` change to `'true'` from `true` along the way from the `.env` file to here?

File: .env.common-local (hunk 1 above)
Comment (bobsilverberg) [reply .env.common-local:31]: I added it here as a clue to a developer that this is where you'd want to change it if you want to affect your local environment. I suppose someone could just edit `.env` instead (I think), but this is where we encourage people to adjust their local environment variables.

File: src/tracking.tsx (hunk 2 above)
Comment (bobsilverberg) [reply src/tracking.tsx:49]: I think it's just the nature of how `process.env` works. It always returns strings to us.

**Pull request merged**